sdk/storage/storage-blob/rollup.base.config.js

```diff
@@ -57,8 +57,11 @@ export function nodeConfig(test = false) {
     onwarn(warning, warn) {
       if (warning.code === "CIRCULAR_DEPENDENCY") {
         throw new Error(warning.message);
+      } else if (warning.message.search("'RequestPolicyOptions' is imported from external module '@azure/core-http' but never used") >= 0) {
+        // Ignoring since this type is imported to be re-exported in Pipeline.ts even it is not used
+      } else {
+        warn(warning);
```

Comment:
I thought about the reverse condition on `warn(warning)` but feel the above code is more explicit about skipping and allow a comment at better place.
```
else if (warning.message.search("'RequestPolicyOptions' is imported from external module '@azure/core-http' but never used") === -1) {
    warn(warning);
}
```